xmake/toolchains/ndk/load.lua

```diff
@@ -174,9 +176,9 @@ function main(toolchain)
             toolchain:add("cflags",  "--sysroot=" .. ndk_sysroot)
             toolchain:add("cxxflags","--sysroot=" .. ndk_sysroot)
             toolchain:add("asflags", "--sysroot=" .. ndk_sysroot)
-            toolchain:add("cflags",  "-isystem " .. path.join(ndk_sysroot, "usr", "include", triple))
-            toolchain:add("cxxflags","-isystem " .. path.join(ndk_sysroot, "usr", "include", triple))
-            toolchain:add("asflags", "-isystem " .. path.join(ndk_sysroot, "usr", "include", triple))
+            toolchain:add("cflags",  "-isystem " .. format("%s/%s/%s/%s", ndk_sysroot, "usr", "include", triple))
```

Comment:
please use path.unix

Reply:
so `path.unix(path.join(...))` instead of just formatting using unix sep? 😕